routes/HomePage.vue

```diff
@@ -134,6 +135,9 @@ export default {
       loadingFilteredCredentials,
       profiles
     };
+  },
+  mounted() {
```

Comment:
Import `onMounted` instead: https://vuejs.org/api/composition-api-lifecycle.html#onmounted

Reply:
Added `onMounted` here: https://github.com/digitalbazaar/bedrock-vue-wallet/pull/32/commits/f3117247a1951842424c3188a0256fc56de58d8d

retested in browser and works (previously tested this on mobile and was working there too)